clients/graphql/models.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type GetAlertsResponse struct {
-	Alerts GetAlertResponseItem
+	Alerts GetAlertResponseItem `json:"alerts0"`
```

Comment:
Why do we want to name the JSON tag like this?

Reply:
outdated pr - sorry